src/main.cc

```diff
@@ -1,21 +1,29 @@
 #include <iostream>
 #include <memory>
 
-#include "gmsh.h"
+#include "GMSH.cc"
+#include "io.h"
 
 int main(int argc, char** argv) {
   try {
     if (argc != 3) {
-      std::cout << "Usage: ./mpm-point-generator /path/to/mesh_file.msh\t"
-                   "Dimension 2 or 3\n";
+      std::cout << "Usage: ./mpm-point-generator /path/to/mesh_file.msh\n"
+                   "Dimenssion 2 or 3\n";
       throw std::runtime_error("Incorrect number of input arguments");
     }
-
+    //! Get dimension or Tdim
     int dimension = std::atoi(argv[2]);
-    std::string filename = argv[1];
 
+    //! Store input/output options including directory
+    std::string inputfile = argv[1];
+    std::string output_point = "../bin/vertexcheck.txt";
```

Comment:
Don't hard code paths. Why are they in `main`. They should be initialised, if at all, in the `IO` class.